README.md

```diff
@@ -1,10 +1,24 @@
-# Nuxt 3 Minimal Starter
+<img src="./src/assets/logo.png" height="200" title="Sigma Dao" />
 
-We recommend to look at the [documentation](https://v3.nuxtjs.org).
+# Sigma Dao
+
```

Comment:
Should we add "beta software" warning, @robert-zaremba ?

Reply:
Yes